src/odemis/acq/stitching/_registrar.py

```diff
@@ -81,14 +79,11 @@ def __init__(self):
         # each tile
         # initialize grid to 1x1. The size will increase as new tiles are
         # added.
-        self.nx = 1
-        self.ny = 1
-        self.coords_ver = [[None]]
-        self.coords_hor = [[None]]
-        self.shifts = [[None]]
+        self.nx = 1  # total number of cols
+        self.shifts_ver = [[None]]
```

Comment:
Please indicate the standard content of each of the variables. `None or 2 floats`? `None or 2 ints`?